src/coreclr/nativeaot/Runtime/threadstore.cpp

```diff
@@ -159,51 +160,47 @@ void ThreadStore::DetachCurrentThread(bool shutdownStarted)
         return;
     }
 
+    // unregister from OS notifications
+    // this can return false if detach notification is spurious and does not belong to this thread.
```

Comment:
```suggestion
    // Unregister from OS notifications
    // This can return false if detach notification is spurious and does not belong to this thread.
```